src/main/scala/io/scalac/wtf/domain/Main.scala

```diff
@@ -0,0 +1,57 @@
+package io.scalac.wtf.domain
+
+import slick.driver.H2Driver.api._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import slick.driver.H2Driver._
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import spray.json.DefaultJsonProtocol._
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import UserService.createUser
+import cats.data.Validated.{Invalid, Valid}
+import cats.implicits._
+import io.scalac.wtf.domain.tables.Users
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+case class JSONUser(email: String, password: String)
```

Comment:
This class has nothing to do with JSON so the naming is wrong. It actually represents a http request for creating new user, so more appropriate would be `UserData` or `CreateUserRequest` or `RegisterUserRequest`.